spec/take_away_spec.rb

```diff
@@ -0,0 +1,60 @@
+require_relative "../lib/customer.rb"
+
+
+describe Takeaway do 
```

Comment:
The tests are concise. One small suggestion would be to split the rspec files per class